ui/src/containers/GphlWorkflowParametersDialog.jsx

```diff
@@ -72,7 +72,7 @@ function GphlWorkflowParametersDialog(props) {
     show,
     updatedFormData,
     handleHide,
-    updateGphlWorkflowParameters,
+    updateGphlWorkflowParameters, // eslint-disable-line no-shadow
```

Comment:
Action creator imported and passed as prop. It will be refactored eventually, so no point in renaming now.